<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Weather</class>
 <widget class="QDialog" name="Weather">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/Resources/weather.jpeg);</string>
  </property>
  <widget class="QLabel" name="City">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>城市</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="CityEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>抚州</string>
   </property>
  </widget>
  <widget class="QLabel" name="Temperature">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>温度</string>
   </property>
  </widget>
  <widget class="QLabel" name="WindPower">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>风力</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TempEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="WeatherStatus">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>48</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>天气状态</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WeatherEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>162</width>
     <height>22</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;哈哈哈&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="WindEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>162</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="GetWeather">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取天气</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SyncWeather">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>同步天气</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
